src/_locales/en_US/messages.json

```diff
@@ -403,6 +403,10 @@
         "message": "Disable sending web addresses you visit to Google. This disables suggestions for similar pages when a page can't be found.",
         "description": "(Chrome only) Checkbox label found on the general settings page"
     },
+    "options_disable_floc": {
```

Comment:
Keeping the same string ID in the hope of easing translation (the previous translation for FLoC would then be there in Transifex translation history).